src/components/Toast/Toast.tsx

```diff
@@ -0,0 +1,43 @@
+import { FC, PropsWithChildren } from "react"
+
+import clsxm from "@/lib/clsxm"
+
+import { FortIconClose } from "@/icons"
+
+export type ToastComponentProps = {
+  isVisible: boolean
+  onDismiss?: () => void
+  className?: string
+}
+
+export const Toast: FC<PropsWithChildren<ToastComponentProps>> = ({
+  className,
+  isVisible,
+  onDismiss,
+  children,
+}) => {
+  return (
+    <div
+      className={clsxm(
+        `${
+          isVisible ? "animate-enter opacity-100" : "animate-leave opacity-0"
+        } w-full max-w-xs rounded-md bg-gray-800 p-4 text-gray-400 shadow`,
+        className
+      )}
+    >
+      {children}
+      {!!onDismiss && (
+        <button
+          onClick={onDismiss}
+          className={clsxm(
+            "ml-auto inline-flex h-7 w-7 rounded-md bg-gray-800 p-1.5 text-gray-500 hover:bg-gray-700 hover:text-white focus:ring-2 focus:ring-gray-300",
```

Comment:
I can't imagine `ml-auto` does anything here?

Reply:
the ml-auto here is used to place the close button (otherwise the btn goes to the left)